docs/user-guide/dictionaries-and-ontologies/loading-new-ontology/skos-dictionary.md

```diff
@@ -0,0 +1,62 @@
+# Dictionaries in SKOS namespace
+
+ODM also supports dictionaries in [SKOS](https://www.w3.org/2009/08/skos-reference/skos.html) namespace. This feature is
+useful when you
+want to organize terms in a hierarchical structure, which enables more advanced search capabilities. Both RDF/XML and
+TURTLE formats
+of dictionaries are supported. Predicated `skos:broader`, `skos:narrower` `skos:related`, `skos:definition`,
+and `skos:exactMatch` are recognized
+and utilized by ODM.
+
+## Example of SKOS dictionary
+
+Here is an example of a short SKOS dictionary in TURTLE format:
+
+<details>
+<summary>TURTLE format</summary>
+
+```ttl
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+<http://vocabulary.boehringer-ingelheim.com/BodySystem/562>
+	a skos:Concept ;
+	skos:prefLabel "Cerebral cortex"@en .
+	
+<http://vocabulary.boehringer-ingelheim.com/BodySystem/104>
+	a skos:Concept ;
+	skos:narrower <http://vocabulary.boehringer-ingelheim.com/BodySystem/562> ;
+	skos:prefLabel "Brain"@en .
+	
+<http://vocabulary.boehringer-ingelheim.com/BodySystem/562> skos:broader <http://vocabulary.boehringer-ingelheim.com/BodySystem/104> .
+```
+</details>
+
```

Comment:
And here is an example of the same dictionary but in RDF/XML format:

Reply:
Slightly rephrased